### DISCUSSION TITLE
Account Flagged

### DISCUSSION THREAD

**Discussion by SJTriedCoding** [community/community#128044] [New to GitHub]
My account has been flagged for no reasons given, I didn't receive any mail or message regarding this. Now I am not able to add people to my repo or someone else cant add me in their repo. I have submitted a ticket also regarding this and its been more than a week and no response from them. Please someone help, the customer support is so bad.

**Comment by VargaFelixErvin** [comment]
Resend/Update Your Support Ticket
You mentioned you submitted a ticket over a week ago. I suggest:
Replying to the original support thread again (don’t submit a new one).
Make sure you selected “Account Flagged” as the issue category in GitHub Support.
Include your GitHub username, brief explanation, and links to any affected repos (yours or ones you're trying to join).

for example:

Hello GitHub Support,

My account has been flagged without any explanation. I haven't received any communication regarding the reason. This is preventing me from collaborating on repositories (I cannot add people or be added to others’ repos).

I submitted a support ticket over a week ago but haven't received any response. I kindly ask for an update on the status of my account review.

My GitHub username: [your-username]
Affected repos (if applicable): [link-to-repo]

Thank you,
[your name]
